dependencies.yaml

```diff
@@ -103,6 +104,7 @@ files:
     includes:
       - common_build
       - depends_on_libcugraph
+      - depends_on_libcuvs
```

Comment:
Wheel builds shouldn't depend on libcuvs libraries, it should be fetched and built statically by CMake.
```suggestion
```